src/InstrumentationEngine.NuGet/InstrumentationEngine.Nuget.csproj

```diff
@@ -5,8 +5,8 @@
   <PropertyGroup>
     <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
     <TargetExt>.nupkg</TargetExt>
-    <NugetSuffix Condition="'$(PlatformOS)'=='Linux'">$(Platform)_$(LinuxDistro)</NugetSuffix>
-    <NugetSuffix Condition="'$(NugetSuffix)'==''">$(Platform)</NugetSuffix>
+    <NugetSuffix Condition="'$(PlatformOS)'=='Linux'">$(Platform.ToLower())_$(LinuxDistro)</NugetSuffix>
```

Comment:
If this and PackageId were moved down to the second PropertyGroup, could we use `$(ArchitectureFolderName)` instead of special casing the platform in certain places with `$(Platform.ToLower())`?

Reply:
Sure, will fix